components/dictionary.txt

```diff
@@ -1,11 +1,133 @@
+actin
+ADT
+al
 al.
+AML
+AnaLysis
+ASXL
+Azacytidine
+arounds
+Baumer
+Benjamini
+bioconductor
+bioinformatics
+Brems
+CCDL
+CCDL's
+cheatsheet
+cheatsheets
+ChIP
+Cmd
+ColorBrewer
+ComplexHeatmap
+CREB
+Crouser
+cytosolic
+Danio
 dataset
 dataset's
+decitabine
 DESeq
 DESeq2
+DESeqDataset
+devtools
+DGE
+directionality
+DocToc
+Dorsoventral
+ECM
+edgeR
+edgeR's
+eg
+ENSDARG
+Ensembl
+ENSG
+Entrez
+et
 et.
```

Comment:
Should not need both. But also, "et." is likely to be wrong: "et" is a word on its own, not an abbreviation.
```suggestion
et
```